test/simulator-e2e-specs.js

```diff
@@ -260,6 +260,36 @@ function runTests (deviceType) {
       (await sim.hasCalendarAccess(BUNDLE_ID)).should.be.false;
     });
 
+    it('should properly start simulator in headless mode on Xcode9+', async function () {
+      const sim = await getSimulator(udid);
+      if (!(parseInt(sim.constructor.name.replace(/\D/g, ''), 10) >= 9)) {
```

Comment:
Yes. We already have `appium-xcode` as a dependency. So you can just use something like:
```js
import xcode from 'appium-xcode';

// ...

before(async function () {
  let xcodeVersion = await xcode.getVersion(true);
  if (xcodeVersion.major < 9) return this.skip();
});
```